core/src/main/scala/org/apache/spark/SSLOptions.scala

```diff
@@ -131,8 +132,9 @@ private[spark] case class SSLOptions(
   /** Returns a string representation of this SSLOptions with all the passwords masked. */
   override def toString: String = s"SSLOptions{enabled=$enabled, port=$port, " +
       s"keyStore=$keyStore, keyStorePassword=${keyStorePassword.map(_ => "xxx")}, " +
+      s"keyPassword=${keyPassword.map(_ => "xxx")}, keyStoreType=$keyStoreType, " +
```

Comment:
nit: we could add `needClientAuth` as well if we would like to dump all.

Reply:
Yeah, added to make it complete.